### PR TITLE
Reduce text steps

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
Waiting on reduce image steps to land

**Comment by k88hudson** [comment]
This opens the text editor immediately; it doesn't start with the keyboard open though, since I'm having trouble getting that to work. Will do in follow-up

File: src/pages/element/text-editor.jsx [hunk 1]
```diff
@@ -54,6 +54,12 @@ var TextEditor = React.createClass({
     }
 
   },
+  componentDidMount: function () {
+    if (window.Platform && window.Platform.openKeyboard) {
+      window.Platform.openKeyboard();
```
Comment (gvn) [inline src/pages/element/text-editor.jsx:60]: Since this doesn't exist do you want to add a TODO comment?

**Comment by gvn** [comment]
Nice! Might be cool to start with an empty text box when you do get the keyboard to come up. I'm assuming just calling `focus` on the input doesn't work?

**Pull request merged**